pkg/daemons/agent/agent_test.go

```diff
@@ -0,0 +1,45 @@
+package agent
+
+import (
+	"testing"
+
+	"github.com/rancher/k3s/pkg/util"
+)
+
+func TestAddFeatureGate(t *testing.T) {
```

Comment:
Test should go into the same module as the thing it's testing, so this should be `pkg/util/gates_test.go`

Reply:
Good call. I missed that.

Reply:
Yeah, thanks for catching that. I missed it in the refactor.